resources/Login.py

```diff
@@ -0,0 +1,38 @@
+from werkzeug.security import check_password_hash
+from flask_restful import Resource
+from flask import request
+from middleware.login_queries import login_results, create_session_token
+
+
+class Login(Resource):
+    def __init__(self, **kwargs):
+        self.psycopg2_connection = kwargs["psycopg2_connection"]
+
+    def post(self):
+        """
+        Login function: allows a user to login using their email and password as credentials
+        The password is compared to the hashed password stored in the users table
+        Once the password is verified, an API key is generated, which is stored in the users table and sent to the verified user
+        """
+        try:
+            data = request.get_json()
+            email = data.get("email")
+            password = data.get("password")
+            cursor = self.psycopg2_connection.cursor()
+
+            user_data = login_results(cursor, email)
```

Comment:
@mbodeantor Can we error here if the user does not exist in DB? Currently it's returning `200` with a `null` response.